javagen/src/main/java/com/azure/autorest/template/ClientMethodTemplateBase.java

```diff
@@ -205,17 +206,34 @@ private static void bodySchemaJavadoc(IType type, JavaJavadocComment commentBloc
             } else {
                 commentBlock.line(indent + appendOptionalOrRequiredAttribute(isRequired, isRootSchema) + "{");
             }
-            bodySchemaJavadoc(((MapType) type).getValueType(), commentBlock, nextIndent, "String", typesInJavadoc, isRequired, false);
+            bodySchemaJavadoc(((MapType) type).getValueType(), commentBlock, nextIndent, "String", typesInJavadoc, isRequired, false, true);
             commentBlock.line(indent + "}");
         } else {
+            String javadoc;
+            if (isInRequestBody) {
+                javadoc = convertToRequestBodySchemaJavadoc(type);
+            } else {
+                javadoc = type.toString();
+            }
```

Comment:
I think we don't need to distinguish whether the Flux in body or not.

I kind of not able to think a way that it is not in body actually.

Reply:
ok